tests/unit/test_messages_integration.py

```diff
@@ -358,7 +365,10 @@ def test_progressbar_brief_terminal(capsys, monkeypatch):
         Line("Uploading stuff [████████████████████████       ] 1400/1788", permanent=False),
         Line("Uploading stuff [███████████████████████████████] 1788/1788", permanent=False),
         Line("Uploading stuff (<---)", permanent=False),
-        Line("And so on", permanent=True),
+        Line("And so on", permanent=False),
```

Comment:
Duplicate this one too, please, as suggested above.

Reply:
Created a ```permanent=True``` version to test the case I previously removed.